packages/playwright-core/src/server/browserType.ts

```diff
@@ -180,7 +180,7 @@ export abstract class BrowserType extends SdkObject {
       if (!registryExecutable || registryExecutable.browserName !== this._name)
         throw new Error(`Unsupported ${this._name} channel "${options.channel}"`);
       executable = registryExecutable.executablePathOrDie(this.attribution.playwright.options.sdkLanguage);
-      await registryExecutable.validateHostRequirements(this.attribution.playwright.options.sdkLanguage);
+      await registry.validateHostRequirementsForExecutablesIfNeeded([registryExecutable], this.attribution.playwright.options.sdkLanguage, /* updateValidationTimestamp */ false);
```

Comment:
I think thats how we discussed it in the meeting according to the notes:

> - add warning during npx playwright install
> - add verify label during validation, otherwise validate again in some flag file.

Reply:
Addressed.